server/metrics.go

```diff
@@ -24,7 +24,7 @@ var (
 			Name: "error_total",
 			Help: "Number of times we have seen a specific error within a specific error domain.",
 		},
-		[]string{"domain"},
+		[]string{},
```

Comment:
Due to the simplified error handling there is no domain anymore, which came from go-kit. So we do not have this value for error metrics anymore.